### PR TITLE
Issue #1609 Added Cache Size to Status

### PR DESCRIPTION
**Pull request by rhnvrm**
Adds cache size to status. #1609

### CODE REVIEW

File: cmd/minishift/cmd/status.go [hunk 1]
```diff
@@ -87,7 +92,20 @@ func runStatus(cmd *cobra.Command, args []string) {
 		diskUsage = fmt.Sprintf("%s of %s", diskUse, diskSize)
 	}
 
-	status := Status{vmStatus, profileName, openshiftStatus, diskUsage}
+	cacheDir := filepath.Join(constants.GetMinishiftHomeDir(), "cache")
+    var size int64
```
Comment (gbraad) [inline cmd/minishift/cmd/status.go]: Not fomatted? Please run `make fmt` before pushing a PR

File: cmd/minishift/cmd/status.go [hunk 2]
```diff
@@ -37,13 +39,15 @@ var statusFormat = `Minishift:  {{.MinishiftStatus}}
 Profile:    {{.ProfileName}}
 OpenShift:  {{.ClusterStatus}}
 DiskUsage:  {{.DiskUsage}}
+CacheUsage: {{.CacheUsage}} Bytes
```
Comment (gbraad) [inline cmd/minishift/cmd/status.go]: We would like to use the HumanReadable form as used in other places in our codebase.

**Comment by gbraad** [comment]
add to whitelist

File: cmd/minishift/cmd/status.go (hunk 2 above)
Comment (rhnvrm) [reply cmd/minishift/cmd/status.go]: Thanks for the review, I will update and push the code to have MB

**Comment by rhnvrm** [comment]
Now the output of `minishift status` looks like:
```
Minishift:  Stopped
Profile:    minishift
OpenShift:  Stopped
DiskUsage:  Unknown
CacheUsage: 541.59 MB
```

File: cmd/minishift/cmd/status.go [hunk 3]
```diff
@@ -87,7 +91,21 @@ func runStatus(cmd *cobra.Command, args []string) {
...
+		if !info.IsDir() {
+			size += info.Size()
+		}
+		return err
+	})
+	if err != nil {
+		atexit.ExitWithMessage(1, fmt.Sprintf("Error finding size of cache: %s", err.Error()))
+	}
+	sizeMB := float64(size) / 1024.0 / 1024.0
+	cacheUsage = fmt.Sprintf("%.2f MB", sizeMB)
```
Comment (gbraad) [inline cmd/minishift/cmd/status.go]: Please use something like https://github.com/minishift/minishift/blob/2e48c0a652fba570829b842473dd9c42ddb48dbc/cmd/minishift/cmd/start.go#L363

**Comment by rhnvrm** [comment]
Thanks, I agree with that, I have pushed a commit with conversion now being performed by the library.

File: cmd/minishift/cmd/status.go (hunk 3 above)
Comment (rhnvrm) [reply cmd/minishift/cmd/status.go]: Have updated

**Comment by gbraad** [comment]
Thank you... output gives the same results?

**Comment by rhnvrm** [comment]
Yes, it gives
```
Minishift:  Stopped
Profile:    minishift
OpenShift:  Stopped
DiskUsage:  Unknown
CacheUsage: 567.9 MB
```

**Comment by gbraad** [comment]
I'll run a test tomorrow, but so far this LGTM. Very minimal change ;-)

Please squash all into a single commit...

**Comment by rhnvrm** [comment]
@gbraad - Thanks! 

Have squashed and force pushed to the branch (y)

File: cmd/minishift/cmd/status.go [hunk 4]
```diff
@@ -37,13 +39,15 @@ var statusFormat = `Minishift:  {{.MinishiftStatus}}
 Profile:    {{.ProfileName}}
 OpenShift:  {{.ClusterStatus}}
 DiskUsage:  {{.DiskUsage}}
+CacheUsage: {{.CacheUsage}}
```
Comment (praveenkumar) [inline cmd/minishift/cmd/status.go]: :+1:

File: cmd/minishift/cmd/status.go [hunk 5]
```diff
@@ -87,7 +92,21 @@ func runStatus(cmd *cobra.Command, args []string) {
...
+		if !info.IsDir() {
+			size += info.Size()
+		}
+		return err
+	})
+	if err != nil {
+		atexit.ExitWithMessage(1, fmt.Sprintf("Error finding size of cache: %s", err.Error()))
+	}
+
+	cacheUsage = units.HumanSize(float64(size))
```
Comment (praveenkumar) [inline cmd/minishift/cmd/status.go]: :+1:

**Comment by amitkrout** [comment]
@praveenkumar Sure, will check it on win10 and update

**Comment by amitkrout** [comment]
@rhnvrm @praveenkumar Tested the pr in different condition on ```win10 + Hyper-V + powershell```. For example m start (fresh) -> m status, m stop -> m status and m start -> m status.

I have little concern on how ```CacheUsage``` is retrieved if vm is stopped  
```
PS C:\Users\CDKQE\go\bin> .\minishift.exe start
PS C:\Users\CDKQE\go\bin> .\minishift.exe status
Minishift:  Running
Profile:    minishift
OpenShift:  Running (openshift v3.9.0+d0f9aed-12)
DiskUsage:  9% of 17.9G
CacheUsage: 418.1 MB
```
```
PS C:\Users\CDKQE\go\bin> .\minishift.exe stop
Stopping local OpenShift cluster...
Cluster stopped.
PS C:\Users\CDKQE\go\bin> .\minishift.exe status
Minishift:  Stopped
Profile:    minishift
OpenShift:  Stopped
DiskUsage:  Unknown
CacheUsage: 538.8 MB
```
```
PS C:\Users\CDKQE\go\bin> .\minishift_pull.exe start
-- Starting profile 'minishift'
[...]
PS C:\Users\CDKQE\go\bin> .\minishift.exe status
Minishift:  Running
Profile:    minishift
OpenShift:  Running (openshift v3.9.0+d0f9aed-12)
DiskUsage:  9% of 17.9G
CacheUsage: 538.8 MB
```

**Comment by gbraad** [comment]
It probably needed some time to finish the image caching, but the results
are as expected.

-- 

   Gerard Braad | http://gbraad.nl
   [ Doing Open Source Matters ]

**Comment by LalatenduMohanty** [comment]
The string `CacheUsage: 418.1 MB` sounds confusing to me. Let see if I can find a better string quickly.

**Comment by rhnvrm** [comment]
@LalatenduMohanty - Have pushed the changes

**Comment by LalatenduMohanty** [comment]
The CI has passed last time and the last change was a trivial change. Hence merging it without waiting for CentOS CI to pass.

**Pull request merged**